README.md

```diff
@@ -51,13 +51,24 @@ rather than IP numbers in a Hyper-V environment. On CentOS systems,
 this requires the bind-utils package (updatedns requires the 'dig'
 program). The bind-utils package isn't otherwise necessary.
 
+- Install minimum version of Ruby (the system must have 1.9 or higher, but 2.2.6 is recommended)
```

Comment:
From what I remember 2.2.6 was selected because of ssl compatibility issue, if that is the case then should not that be a specific ruby version as BASERUBY?